### ISSUE TITLE
Reach out to Notre Dame about migration

### ISSUE THREAD

**Issue by crowesn** [uclibs/scholar_uc#1550]
### Descriptive summary

Ask about migration approach. XML export?

**Comment by scherztc** [comment]
Contacted ND by email.

**Comment by scherztc** [comment]
We had plans to begin migrating to Fedora 4 this summer and fall. However, our Digital Collections Librarian and Metadata Wrangler (Ruth Kitchin Tillman) has left for Penn State, and I'm waiting for my lead developer to transfer from my sibling unit; He's locked into a major website rennovation project until sometime in August. With such major changes, we are pushing back our migration plans.

That said there are a few things I may be able to provide guidance on:

1) Nested Collections - I've added some documentation to the Collection Extensions document on the Duraspace Wiki. We have some logic for nested collections in a plugin for CurateND.
2) Data migration - We have added the JSON-LD format to output of CurateND objects (here's the code). We envision using this as our egress from Fedora 3.
3) Batch Ingestor - We have a batch ingester application written in Go. It speaks JSON-LD, CSV, and our own ROF format. From there we can ingest objects into CurateND.

Our general plan is:

1) Update the batch ingester to write to Fedora 4 (with appropriate data mapping)
2) Export the JSON-LD objects from CurateND to a batch ingest ready location / format
3) Run the Batch Ingester to send the objects to Fedora 4
